documentation/user_docs/docs/manual/Extracting_data_from_SQW_files_and_objects.rst

```diff
@@ -0,0 +1,637 @@
+
+.. |Q| replace:: :math:`|\textbf{Q}|`
+
+.. role:: matlab(code)
+   :language: matlab
+
+Normally the whole data produced in neutron experiment are too large to fit the memory 
+of majority of modern computers. Horace have stored them in large 4-D dataset, containing full information about experiment,
+4-Dimensional image of reciprocal space ``hkl-dE`` covered during experiment and neutron events registered during experiment.
+User operates with number of smaller objects, extracted from this dataset using various forms of ``cut`` command.
+
+``cut`` is probably the most important Horace command, so its parameters demand special attention.
+
+cut
+===
+
+``cut`` takes data from an ``sqw`` object, ``dnd`` object or saved ``sqw`` or
+``dnd`` file and converts it to an object of the same or reduced size and dimensions.
+The resulting cut object is itself an ``sqw`` or ``dnd`` object which can be plotted, manipulated,
+etc. like any other ``sqw`` or ``dnd`` object. If the resulting object is itself too big to fit memory,
+it will be backed by appropriate ``sqw`` file. 
+The required inputs are as follows:
+
+.. note::
+
+   For the differences between ``dnd`` and ``sqw`` objects see: :ref:`here
+   <manual/FAQ:What is the difference between sqw and dnd objects>`
+
+.. code-block:: matlab
+
+   my_cut = cut(data_source, proj, p1_bin, p2_bin, p3_bin, p4_bin,varargin)
+
+where ``varargin`` describes optional arguments and may contain:
+
+  ``-nopix``   -- the key requests that the cut should contain just image (``dnd`` object) and drop all pixels contributing into this image. This  option is faster but should not be used if you want to do subsequent cuts from the resulting cut ``my_cut``.
+  
+  ``filename`` -- if provided, requests that the resulting file is backed by the file with the ``filename``. This option is mandatory if cut is called without output argument.
+
+Cutting consists of a rebinning of the pixels into the bins specified by the cut
+parameters (described below).  These binned pixels will make up the ``dnd`` of
+the output object which contains information regarding the plottable data.
+
+
+Data Source
+-----------
+
+``data_source`` in ``cut`` is either a string giving the full filename (including path) of
+the input ``.sqw`` file or just the variable containing an ``sqw`` or ``dnd``
+object stored in memory from which the pixels will be taken.
+
+.. warning::
+
+   By convention both ``sqw`` and ``dnd`` objects use the ``.sqw`` suffix when
+   stored to disc. It is advisable to name the file appropriately to distinguish
+   the types stored inside, e.g. ``MnO2_sqw.sqw``, ``MnO2_d2d.sqw``.
+
+Projection (proj)
+-----------------
+
+Second argument defines the coordinate system and thus the meaning of the 
+`Binning arguments`_ you provided as arguments to `cut` and will 
+use to plot and analyse the data.
+
+``proj`` should be an child of a ``aProjectionBase`` class (such as ``line_proj``,
```

Comment:
A lot of this goes into internal Horace representations which should not concern users. This should not be in the mnal